arch/risc-v/src/mpfs/mpfs_opensbi.c

```diff
@@ -22,25 +22,12 @@
  * Included Files
  ****************************************************************************/
 
-#include <nuttx/config.h>
-#include <assert.h>
-#include <errno.h>
-#include <stdint.h>
-#include "riscv_arch.h"
-
 #include <hardware/mpfs_plic.h>
 #include <hardware/mpfs_memorymap.h>
 #include <hardware/mpfs_clint.h>
 #include <hardware/mpfs_sysreg.h>
 
-/* OpenSBI will also define NULL. Undefine NULL in order to avoid warning:
- * 'warning: "NULL" redefined'
- */
-
-#ifdef NULL
-  #undef NULL
-#endif
-
+#include <sbi/riscv_io.h>
```

Comment:
Probably after this PR I will try to renew my changes with OpenSBI NuttX types header file.